examples/tutorial 1a - Using a Quantum Device to solve MIS.ipynb

```diff
@@ -4,7 +4,34 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# MIS High level"
+    "# Tutorial 1: using a Quantum Device to solve MIS\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The high-level goal of this tutorial is to execute an instance of the classic NP-hard Maximum Independent Set (MIS) problem, using both classical and quantum methods.\n",
```

Comment:
Note: That's sufficient for a tutorial MVP, but at some point, we'll want to tell a more compelling story (e.g. "solving a scheduling problem", "placing satellites", "placing antennas", etc.)

Reply:
Fair.